porting/deploying-rootfs.md

```diff
@@ -0,0 +1,17 @@
+# Deploying the rootfs and system.img
+
+Once you have built the system.img from android tree, you can download and install the rootfs using halium-install script in [halium-scripts repository](https://github.com/Halium/halium-scripts/).
+
+Currently latest rootfs is available at, bshah's personal server: [Link](http://bshah.in/halium/halium-rootfs-20170630-151006.tar.gz)
+
+Use the halium-install script as below, when device is connected in recovery mode
```

Comment:
You can use the halium-install script as below, when your device is connected and in recovery mode.